src/solver/types.jl

```diff
@@ -291,6 +293,16 @@ struct GCMForcing{FT}
     cfsite_number::String
 end
 
+"""
+    ExternalDrivenTVForcing
+    
+Forcing specified by external forcing file and a start date.
+"""
+struct ExternalDrivenTVForcing{FT}
+    external_forcing_file::String
+    start_date::String
```

Comment:
If think we should be able to get rid of this and use directly the simulation start_date.